src/actions/authActions.js

```diff
@@ -271,6 +273,7 @@ export const lockScreenAction = (onLoginSuccess?: Function, errorMessage?: strin
 
 export const logoutAction = () => {
   return async (dispatch: Function) => {
+    Intercom.logout();
```

Comment:
Maybe we should call it when we delete the wallet, and not on logout?

Reply:
I think it's ok as the current flow of wallet delete fires logoutAction.

Reply:
you're right :D we should rename this action one day